src/main/java/com/commercetools/sync/producttypes/utils/AttributeDefinitionsUpdateActionUtils.java

```diff
@@ -153,48 +149,51 @@
                     }
                 ));
 
-        return oldAttributeDefinitions
-            .stream()
-            .map(oldAttributeDefinition -> {
-                final String oldAttributeDefinitionName = oldAttributeDefinition.getName();
-                final AttributeDefinitionDraft matchingNewAttributeDefinitionDraft =
+        final List<UpdateAction<ProductType>> updateActions = new ArrayList<>();
+
+        for (AttributeDefinition oldAttributeDefinition : oldAttributeDefinitions) {
+
+            final String oldAttributeDefinitionName = oldAttributeDefinition.getName();
```

Comment:
what if the name is `null` here?  this should be validated. Where are the unit tests testing these cases?

Reply:
The .map function does not throw an error. I don’t want to surround with try-catch.